source/sessions/driver-sessions.rst

```diff
@@ -979,19 +977,19 @@ Test Plan
 =========
 
 The test plan SHOULD be run against both replica set and sharded cluster
-topologies.  It MUST NOT be run against a standalone server.
```

Comment:
If you're removing the prohibition about running on a standalone, I would venture that the topology really doesn't matter at all. In that case, should the first sentence be changed as well? Perhaps:

> The test plan SHOULD be run against deployments that support sessions

Or just remove this sentence as well.

Reply:
Good idea, I'll just remove it